<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QMainWindow" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>218</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="peersLabel">
         <property name="text">
          <string>Connected peers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="peerList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="gamesLabel">
         <property name="text">
          <string>Peers waiting for opponent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="gameList">
         <property name="font">
          <font>
           <family>Segoe UI</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>262</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="restartButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Restart server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Shut down</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiondad">
   <property name="text">
    <string>dad</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
